src/Nancy.Hosting.Aspnet/DefaultNancyAspNetBootstrapper.cs

```diff
@@ -73,7 +73,7 @@ protected override IEnumerable<IRegistrations> GetRegistrationTasks()
         /// </summary>
         /// <param name="context">Current request context</param>
         /// <returns>IEnumerable of INancyModule</returns>
-        public override sealed IEnumerable<INancyModule> GetAllModules(NancyContext context)

```

Comment:
Dude, ReSharper was telling me `inconsistent modifiers declaration order` and it was making me twitch! :smile: